src/streamlink_cli/main.py

```diff
@@ -851,6 +851,9 @@ def setup_options():
         streamlink.set_option("ffmpeg-audio-transcode", args.ffmpeg_audio_transcode)
     if args.ffmpeg_no_start_at_zero:
         streamlink.set_option("ffmpeg-start-at-zero", False)
+    if args.ffmpeg_copyts:
+        streamlink.set_option("ffmpeg-copyts", True)
+        streamlink.set_option("ffmpeg-start-at-zero", False)
```

Comment:
I'm not entirely sure, but I think it makes more sense overriding the CLI argument value of `args.ffmpeg_no_start_at_zero` (setting it to `True`) and moving the `if args.ffmpeg_copyts` block up. Then the session option `ffmpeg-start-at-zero` does only get set in one single place and it aligns with the argument help text which should be referring to the other CLI argument.

Reply:
@beardypig, do you agree with the proposal of bastimeyer, I quarrel with the proposal?